src/utils/pager.ts

```diff
@@ -1,8 +1,8 @@
 import store, {
   updatePagerLastItemIndexMutation,
   updateItemsToShowMutation
-} from "@/store";
-import MarketplaceItemModel from "@/models/marketplaceItemModel";
+} from "./../store";
+import MarketplaceItemModel from "./../models/MarketplaceItemModel";
```

Comment:
Why to use relative path ant not continue using [name module mapper](https://github.com/Kentico/ems-extension-marketplace/blob/master/package.json#L88) ?